docs/proposals/security-context-constraints.md

```diff
@@ -0,0 +1,303 @@
+## Abstract
+
+Security context constraints relate [security contexts](./security_contexts.md) and
+[service accounts](http://docs.k8s.io/design/service_accounts.md).  They are configured for a
+cluster, namespaces, and users within the cluster to govern how a [security context](./security_contexts.md) is established.
```

Comment:
> They are configured for a cluster

Is this the namespace's policy default or an actual extra policy?

>  namespaces

:+1:

> users within the cluster

I realize this is a use case but we I think we should leave this out initially. I'm not sold that it's valuable and it's going to be much easier to add later on then it will be to take away.